internal/cli/doc-gen.go

```diff
@@ -120,6 +131,21 @@ layout: default
 `
 )
 
+type pageData struct {
```

Comment:
**Nit:** It's implied that the struct contains data, might we prefer a more concise and/or descriptive name?

```suggestion
type documentationPage struct {
```

or simply

```suggestion
type page struct {
```

Reply:
Done in 87eada9e4273063d1ce9051aef2fc3a963d631ba